packages/@lwc/integration-karma/test/component/refs/index.spec.js

```diff
@@ -333,10 +335,32 @@ describe('refs', () => {
             expect(elm.getRef('foo')).toBe(elm.shadowRoot.querySelector('x-rerender-element'));
             for (let i = 0; i < 3; i++) {
                 elm.version = i;
+                expect(elm.getRef('foo')).toBe(elm.shadowRoot.querySelector('x-rerender-element'));
                 await Promise.resolve();
                 expect(elm.getRef('foo')).toBe(elm.shadowRoot.querySelector('x-rerender-element'));
             }
         });
+
+        it('element with a different static ref', async () => {
+            const elm = createElement('x-rerender-element-static-ref', {
+                is: RerenderElementStaticRef,
+            });
+            document.body.appendChild(elm);
+
+            await Promise.resolve();
+
+            const fooDiv = elm.getRef('foo');
+            expect(fooDiv).not.toBeUndefined();
+
+            expect(elm.getRef('foo')).toBe(fooDiv);
```

Comment:
I'm not sure I understand the reason behind this assertion. Was `elm.getRef('foo')` returning `undefined` on a second non-async call to `getRef`? It doesn't hurt anything, I just want to understand. It reads like `expect(thing).toBe(thing)`.

Reply:
yeah this was a mistake on my part